script.js

```diff
@@ -13,6 +14,207 @@ function closeNav() {
+  parent.appendChild(btns);
+
+  parentElement.appendChild(parent);
+
+  document.getElementById('closeButton').addEventListener('click', closePopup);
+}
+
+function populateMainpage() {
+  const parentElement = document.getElementById('parentElementId');
+
+  for (let x = 0; x < arrMobile.length; x += 1) {
+    const parent = document.createElement('section');
+    parent.classList.add('works', 'info_mobile');
+    parent.setAttribute('id', 'jump-to-works-section');
+    const childImg = document.createElement('img');
+    childImg.src = arrMobile[x].image;
+    parent.appendChild(childImg);
+
+    const infoDiv = document.createElement('div');
+    infoDiv.classList.add('info');
+    parent.appendChild(infoDiv);
+    const child1 = document.createElement('h1');
+    child1.classList.add('tonic');
+    child1.textContent = arrMobile[x].name;
+
+    infoDiv.appendChild(child1);
+
+    const childUl = document.createElement('ul');
+    childUl.classList.add('stack');
+
+    for (let y = 0; y < arrMobile[x].technology.length; y += 1) {
+      const childLi = document.createElement('li');
+      childLi.textContent = arrMobile[x].technology[y];
+      childLi.classList.add('stack-elem');
+      childUl.appendChild(childLi);
+    }
+
+    infoDiv.appendChild(childUl);
+
+    const childDesc = document.createElement('p');
+    childDesc.textContent = arrMobile[x].description;
+    childDesc.classList.add('pg2');
+    infoDiv.appendChild(childDesc);
+
+    const buttonElement = document.createElement('button');
+    buttonElement.classList.add('see-project');
+
+    buttonElement.textContent = ('See Project ');
+    infoDiv.appendChild(buttonElement);
+
+    buttonElement.setAttribute('data-index', x);
+
+    buttonElement.addEventListener('click', (event) => {
+      const butn = event.target;
+      const projectIndex = butn.getAttribute('data-index');
+      const nameProject = arrMobile[projectIndex].name;
+      const descProject = arrMobile[projectIndex].description;
+      const imageProject = arrMobile[projectIndex].image;
+      const techProject = arrMobile[projectIndex].technology;
+      const link1Project = arrMobile[projectIndex].link1;
+      const link2Project = arrMobile[projectIndex].link2;
+      Popup(nameProject, descProject, imageProject, techProject, link1Project, link2Project);
+    });
+
+    parent.appendChild(infoDiv);
+    parentElement.appendChild(parent);
+  }
+}
+
+document.addEventListener('DOMContentLoaded', populateMainpage);
+
+const form = document.querySelector('#contact-me-form');
+const email = form.querySelector('#e-mail');
+const formSubmitBtn = form.querySelector('.button-form');
+
+function emailLowerCase() {
+  const emailValue = email.value;
+
+  if (emailValue.toLowerCase() === emailValue) {
+    document.getElementById('error-message').style.visibility = 'hidden';
+    return true;
+  }
+
+  document.getElementById('error-message').style.visibility = 'visible';
+  return false;
```

Comment:
- [ ] Just like on mobile, there is an overlap and a lot of adjustments needed to be made to get your design to replicate the figma design given. Kindly review the requirements below, and also compare the designs to see areas to review based on the `CSS` contaier which houses your whole cards and the `popup`.

- You can use [This link](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) to see a sample implementation 

### View:
![image](https://github.com/umarahgul/Portfolio/assets/50990925/e762a0cd-d84a-47bd-9a85-389a534186ea)

### Figma design:
![image](https://github.com/umarahgul/Portfolio/assets/50990925/23c616d5-6c34-4a70-a2e9-d5adc2b5069f)